Source/Core/Common/Crypto/bn.cpp

```diff
@@ -8,21 +8,19 @@
 #include "Common/CommonTypes.h"
 #include "Common/Crypto/bn.h"
 
-static void bn_zero(u8* d, u32 n)
+static void bn_zero(u8* d, std::size_t n)
 {
   memset(d, 0, n);
 }
 
-static void bn_copy(u8* d, const u8* a, u32 n)
+static void bn_copy(u8* d, const u8* a, std::size_t n)
 {
   memcpy(d, a, n);
 }
 
-int bn_compare(const u8* a, const u8* b, u32 n)
+int bn_compare(const u8* a, const u8* b, std::size_t n)
```

Comment:
Can't this entire function be collapsed into a clamped `memcmp`?

Reply:
Done. (my original goal was to clean up ec, not bn :P)